securedrop_client/gui/widgets.py

```diff
@@ -481,7 +481,7 @@ def __init__(self, message, align):
             label.setStyleSheet(label.css + 'border-bottom-left-radius: 0px;')
         layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(layout)
-        self.setContentsMargins(0, 0, 0, 0)
+        self.setContentsMargins(38, 38, 38, 38)
```

Comment:
The effect using `setContentsMargins` has is to increase the spacing between the various UI elements, i.e. for two messages you can compare the view in the image in the issue [here](https://user-images.githubusercontent.com/7832803/48035040-cf678c00-e116-11e8-8556-0e96f6a7ae78.png) with the image below from me trying out this PR:

<img width="1442" alt="screen shot 2019-01-07 at 4 08 51 pm" src="https://user-images.githubusercontent.com/7832803/50800793-a8544480-1296-11e9-9ab0-e0cb2d3ffca4.png">

which is close, but not quite what we want. What we can do is use Qt's QSizePolicy, which you can read more about [here](https://doc.qt.io/qt-5/qsizepolicy.html). To fix this issue we'll then want to do two things:

1. Specify a minimum height for each message bubble using CSS's min-height property. 
2. Set the size policies to indicate that we want messages to be able to expand vertically, but not horizontally. 

Check out one way of doing this here: e0666c0381cf092e2dea96260927a847e803179f. We should get something that looks like for conversations that fill and do not fill the view respectively: 

<img width="722" alt="screen shot 2019-01-07 at 5 47 47 pm" src="https://user-images.githubusercontent.com/7832803/50804803-74801b80-12a4-11e9-82af-c219e8825df4.png">

<img width="704" alt="screen shot 2019-01-07 at 5 48 17 pm" src="https://user-images.githubusercontent.com/7832803/50804809-7944cf80-12a4-11e9-9e48-5c9d5837e220.png">

Play around with this and let me know what you think!